runtime.md

```diff
@@ -99,7 +97,7 @@ This operation MUST return the state of a container as specified in the [State](
 `create <container-id> <path-to-bundle>`
 
 This operation MUST [generate an error](#errors) if it is not provided a path to the bundle and the container ID to associate with the container.
-If the ID provided is not unique across all containers within the scope of the runtime, or is not valid in any other way, the implementation MUST [generate an error](#errors) and a new container MUST NOT be created.
+If the ID provided is not unique across all containers within the [scope of the container](#scope-of-a-container), or is not valid in any other way, the implementation MUST [generate an error](#errors) and a new container MUST NOT be created.
```

Comment:
The [scope of the container](https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc5/runtime.md#scope-of-a-container) we link here, doesn't quite fit this statement, it's scopt of **A** container, if an ID is unique or not, would be compared with other containers', which is out of the scope of **A** container.